mobile-app/pubspec.yaml

```diff
@@ -9,16 +9,11 @@ dependencies:
     git:
       url: https://github.com/Mythar/flutter_font_awesome_web_names
       ref: master
-  cupertino_icons: ^1.0.3
```

Comment:
This package is used for loading icons for iOS devices. As of now we're not primarily working for an iOS app but when we do we'll have to include this package along with [flutter_platform_widgets](https://pub.dev/packages/flutter_platform_widgets) for easier handling of which type of widget to show.